package.json

```diff
@@ -35,5 +35,8 @@
     "env": [
       "jest"
     ]
+  },
+  "dependencies": {
```

Comment:
This should be in `devDependencies`. Ideally when we ship #92, the published module will not have any runtime dependencies.